### PR TITLE
test(tabs): add tests

### PR DESCRIPTION
**Pull request by sofiushko**

### CODE REVIEW

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const Tabs: React.FC<TabsProps> = ({
     size = 'm',
     activeTab,
     allowNotSelected = false,
-    items = [],
+    items,
```
Comment (amje) [inline src/components/Tabs/Tabs.tsx:46]: Looks like a breaking change

File: src/components/Tabs/__tests__/Tabs.test.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+const tabTitle1 = 'Tab 1 title';
+const tab1 = {id: tabId1, title: tabTitle1};
+
+const tabId2 = 'tab2';
+const tabTitle2 = 'Tab 2 title';
+const tab2 = {id: tabId2, title: tabTitle2};
+
+const qaId = 'tabs';
+
+describe.only('Tabs', () => {
```
Comment (amje) [inline src/components/Tabs/__tests__/Tabs.test.tsx:16]: only

File: src/components/Tabs/__tests__/TabsItem.test.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import {render, screen, fireEvent} from '@testing-library/react';
+import {TabsItem} from '../TabsItem';
+import {GearIcon} from '../../icons/GearIcon';
+
+const tabId = 'tab-id';
+const tabTitle = 'Tab title';
+
+describe.only('TabsItem', () => {
```
Comment (amje) [inline src/components/Tabs/__tests__/TabsItem.test.tsx:10]: only

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/Tabs/__tests__/TabsItem.test.tsx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    });
+
+    test('should not call onClick on tab click', async () => {
+        const onClickFn = jest.fn();
+
+        render(<TabsItem id={tabId} title={tabTitle} onClick={onClickFn} />);
+
+        const component = screen.getByRole('tab');
+
+        fireEvent.keyDown(component, {keyCode: 9});
```
Comment (ogonkov) [inline src/components/Tabs/__tests__/TabsItem.test.tsx:165]: can we use `@testing-library/user-event`?

File: src/components/Tabs/__tests__/TabsItem.test.tsx (hunk 3 above)
Comment (ogonkov) [reply src/components/Tabs/__tests__/TabsItem.test.tsx:10]: there is actually no need to wrap all tests in common `describe`, jest is adding test name anyway

```
PASS src/components/Tabs/__tests__/TabsItem.test.tsx (14.139 s)
  TabsItem
    ✓ should render tab item by default (29 ms)
```

File: src/components/Tabs/__tests__/TabsItem.test.tsx (hunk 4 above)
Comment (sofiushko) [reply src/components/Tabs/__tests__/TabsItem.test.tsx]: done)

File: src/components/Tabs/Tabs.tsx (hunk 1 above)
Comment (sofiushko) [reply src/components/Tabs/Tabs.tsx]: This prop is required by types https://github.com/yandex-cloud/uikit/blob/main/src/components/Tabs/Tabs.tsx#L32
Maybe we should make it optional then?

**Review by amje — APPROVED** [review]

**Review by ogonkov — APPROVED** [review]

**Pull request merged**